app/models/application_qualification.rb

```diff
@@ -190,6 +190,20 @@ def required_gcse?
       REQUIRED_GCSE_SUBJECTS.include?(subject)
   end
 
+  # We are doing almost a similar as set public id but forcing generation
+  # of new ids for constituent_grades.
+  # This should be deleted after the incident
```

Comment:
https://trello.com/c/a7rCQrl0/1424-remove-constituent-grade-publicid-generation-code

Reply:
Can we tag this with a TODO and a link to the ticket?

Reply:
Humm, in regards to add Todo's in the code I am in favor of not adding those, I am against adding Todo's in the code (although Apply has some of them).  
Quoting Gandalf "I would not take the road to add Todo's in the code unless I had no other choice." 😅 
Jokes aside, I consider a good practice to avoid using todos in code, as they often lack clarity regarding how they will be addressed. Additionally, they tend to lack prioritization, resulting in tasks that may never be completed.
I think the route of adding a card is enough and I will make sure that this specific card is delivered through.
My view is things like these should only happens in emergency situations like the incident.